lib/styleDictionary/index.js

```diff
@@ -0,0 +1,65 @@
+const { outputHelper } = require('./util');
+const { mapBaseColors, mapTextStyleColors, mapBorderStyleColors, mapShadowStyleColors } = require('./colorMapping');
+const { mapTextStyleAssets } = require('./assetMapping');
+const { mapTextStyleSizes, mapBorderStyleSizes } = require('./sizeMapping');
+
+/**
+ * Maps the colors of a given input of ITP tokens to Style Dictionary compatible color tokens.
+ *
+ * @param Object input: ITP tokens 
+ * @returns Object: a StyleDictionary tree of category 'color' 
+ */
+const mapColors = input => {
+  const colorsOutput = mapBaseColors(input.colors);
+  const fontColorsOutput = mapTextStyleColors(input.textStyles);
+  const borderColorsOutput = mapBorderStyleColors(input.borders);
+  const shadowColorsOutput = mapShadowStyleColors(input.shadows);
+
+  const output = Object.assign(
```

Comment:
Why use Object.assign here when everywhere else we use spread?

Reply:
Old habit ;(